glide.lock

```diff
@@ -1,37 +1,37 @@
-hash: d088a750a1a81b463b0056b81edd3ce849b9bef0eb44861f67041e13b95bade0
-updated: 2016-07-06T16:33:39.979049165+02:00
+hash: bb06d4eb9b26ddb217ba46a9c1924a870e9abbf3259dabffd19fa4407b7d769c
+updated: 2016-10-17T14:22:17.272063593-07:00
 imports:
 - name: github.com/appc/spec
-  version: fd5e3d8f5d596c6fbd506e80b5e13ba8a97b704c
+  version: f329150233aa3ac6c83ab5ba02a2b5a8bd03d365
```

Comment:
I am also not aware of a functionality to update just a single package in glide, unfortunately. That is why we pin versions in rkt to avoid general bumps like this. I suggest to pin versions and bump just the single package, or make a local test-run with unchecked tilde-versions in glide.yaml to check if there are note-worthy updates in other packages too.